tests/utils/test_process_utils.py

```diff
@@ -105,6 +109,20 @@ def test_should_print_all_messages1(self):
 
         assert ["Executing cmd: bash -c 'echo CAT; echo KITTY;'", 'Output:', 'CAT', 'KITTY'] == msgs
 
+    def test_should_print_all_messages_from_cwd(self):
```

Comment:
```suggestion
    def test_should_print_all_messages_from_cwd(self, tmp_path):
```

Reply:
`unittest.TestCase` are only compatible with `autouse` fixture. I have changed a little bit the test so I don't need the `self.assertLogs` method. This way I can remove the extends to `unittest.TestCase` and use normal fixtures.

Let me know if this is better.